ED/protocol.html

```diff
@@ -795,11 +795,9 @@ <h3 property="schema:name">Reading Resources</h3>
                 <div datatype="rdf:HTML" property="schema:description">
                   <p><span about="" id="server-safe-methods" rel="spec:requirement" resource="#server-safe-methods"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> support the HTTP <code>GET</code>, <code>HEAD</code> and <code>OPTIONS</code> methods [<cite><a class="bibref" href="#bib-rfc7231">RFC7231</a></cite>] for clients to read resources or to determine communication options.</span></span> [<a href="https://github.com/solid/specification/issues/39#issuecomment-538017667" rel="cito:citesAsSourceDocument">Source</a>]</p>
 
-                  <p>When responding to authorized requests:</p>
+                  <p><span about="" id="server-allow-methods" rel="spec:requirement" resource="#server-allow-methods"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> indicate the HTTP methods supported by the target resource by generating an <code>Allow</code> header field in successful responses.</span></span></p>
```

Comment:
But does that depend on auth? So 401/403 replies are ok?

Reply:
"successful responses" as in 2xx. 401/403 shouldn't include Allow. Essentially 2xx and 405 (as per RFC) is recommended.

Reply:
Yes. If POST (and other methods) would be a "valid request" supported by the target resource, include it in Allow. See also last three comments at https://github.com/solid/specification/pull/353 .